docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -29,23 +29,43 @@ APM Server exposes an HTTP endpoint, and as with anything that opens ports on yo
 you should be careful about who can connect to it.
 Firewall rules are recommended to ensure only authorized systems can connect.
 
-[float]
 [[apm-feature-roles]]
-=== Feature roles
+=== Create and assign feature roles to users
+
+++++
+<titleabbrev>Use feature roles</titleabbrev>
+++++
 
-You can use role-based access control to grant users access to secured
-resources. The roles that you set up depend on your organization's security
-requirements and the minimum privileges required to use specific features.
+It's recommended that you only grant users the minimum privileges required to use specific features. One approach to managing access on a feature-by-feature basis is to create several feature-related _roles_ and assign one or more of these roles to each _user or group_ based on which features the user needs to access.
 
-Typically, you need to create the following separate roles:
+[TIP]
+====
+A _role_ identifies a set of permissions that translate to privileges on resources. You can associate a _user or group_ with an arbitrary number of roles. The total set of permissions that a user has is defined by the union of the permissions in all its roles.
+====
 
-* <<apm-privileges-to-publish-events,Writer role>>: To publish events collected by APM Server.
-* <<apm-privileges-to-publish-monitoring,Monitoring role>>: One for sending monitoring
-information, and another for viewing it.
-* <<apm-privileges-api-key,API key role>>: To create and manage API keys.
-* <<apm-privileges-agent-central-config,Central configuration management role>>: To view
-APM Agent central configurations.
-* <<apm-privileges-rum-source-mapping,RUM source mapping role>>: To read RUM source maps.
+Below are some common tasks that users in your organization might need to do and links to more information on creating roles that provides access to the right resources:
+
+* <<apm-privileges-agent-central-config,*Central configuration management role*>>:
+Allows a user to view APM Agent central configurations, which is *required* when
+<<apm-agent-configuration,central configuration management>> is enabled (it is enabled by default).
+* <<apm-privileges-to-publish-events,*Writer role*>>:
```

Comment:
The writer role is actually the most important role, which allows apm-server to write to ES (although users may configure apm-server to write to something else e.g. logstash). I wonder if we should move it to the top and mark it as required as well.